client/webserver/site/src/html/markets.tmpl

```diff
@@ -182,16 +182,17 @@
                     <div class="mt-2 fs14 text-right" id="orderPreview"></div>
 
                     {{- /* TIME-IN-FORCE CHECK BOX */ -}}
-                    <div class="mt-2 text-left pl-4" id="tifBox">
+                    <div class="my-1 text-left pl-4" id="tifBox">
                       <input id="tifNow" class="form-check-input" type="checkbox" value="">
                       <label class="form-check-label" for="tifNow">
-                        Match for one epoch only
+                        Immediate or cancel
+                        <span class="ico-info fs12" data-tooltip="If the order doesn't fully match during the next match cycle, any unmatched quantity will be canceled."></span>
```

Comment:
Slightly hesitant about the "cancel" wording here because the order status will be executed if there's no match or it just doesn't go on the books.

Reply:
Could we work with a variation of "Do not allow to become booked" or "Taker-only"?